website/data/docs-nav-data.json

```diff
@@ -989,6 +989,10 @@
       {
         "title": "Token Helpers",
         "path": "commands/token-helper"
+      },
+      {
+        "title": "<code>hcp</code>",
```

Comment:
The list of commands is alphabetized, so the `hcp` entry should be after `events`.